src/pages/registration/references_tab/components/IsbnListField.tsx

```diff
@@ -17,47 +23,49 @@ const IsbnListField = () => {
   return (
     <Field name={ReferenceFieldNames.ISBN_LIST}>
       {({ field, form: { setFieldValue, setFieldTouched }, meta: { error } }: FieldProps) => (
-        <Autocomplete
-          freeSolo
-          autoSelect
-          multiple
-          options={[]}
-          value={field.value ?? ''}
-          onChange={(_: ChangeEvent<unknown>, value: string[], reason) => {
-            setFieldTouched(field.name);
-            if (reason === 'create-option' || reason === 'blur') {
-              const newIsbn = value.pop()?.trim().replace(/-/g, '');
-              if (newIsbn?.match(isbnRegex)) {
-                setFieldValue(field.name, [...value, newIsbn]);
+        <MuiThemeProvider theme={lightTheme}>
+          <Autocomplete
+            freeSolo
+            autoSelect
+            multiple
+            options={[]}
+            value={field.value ?? ''}
+            onChange={(_: ChangeEvent<unknown>, value: string[], reason) => {
+              setFieldTouched(field.name);
+              if (reason === 'create-option' || reason === 'blur') {
+                const newIsbn = value.pop()?.trim().replace(/-/g, '');
+                if (newIsbn?.match(isbnRegex)) {
+                  setFieldValue(field.name, [...value, newIsbn]);
+                } else {
+                  dispatch(setNotification(ErrorMessage.INVALID_ISBN, NotificationVariant.Warning));
+                }
               } else {
-                dispatch(setNotification(ErrorMessage.INVALID_ISBN, NotificationVariant.Warning));
+                setFieldValue(field.name, value);
               }
-            } else {
-              setFieldValue(field.name, value);
+            }}
+            renderTags={(value: string[], getTagProps) =>
+              value.map((option: string, index: number) => (
+                <Chip
+                  data-testid="isbn-chip"
+                  label={option}
+                  {...getTagProps({ index })}
+                  color={error && error[index] ? 'secondary' : 'default'}
+                />
+              ))
             }
-          }}
-          renderTags={(value: string[], getTagProps) =>
-            value.map((option: string, index: number) => (
-              <Chip
-                data-testid="isbn-chip"
-                label={option}
-                {...getTagProps({ index })}
-                color={error && error[index] ? 'secondary' : 'default'}
+            renderInput={(params) => (
+              <TextField
+                {...params}
+                data-testid="isbn-input"
+                label={t('references.isbn')}
+                helperText={<StyledHelperText>{t('references.isbn_helper')}</StyledHelperText>}
```

Comment:
Syns egentlig ikke denne løsningen ble sånn kjempebra heller, for her må vi likevel sette fargen manuelt til hvit , siden `helperText` er en del av TextField (ligthTheme), men den vises utenfor boksen (darkTheme). så dette blir jo feil igjen hvis vi skal bruke denne komponenten men lys bakgrunn 😭

Reply:
(Sett hvit label i darkTheme igjen?)

Reply:
Umiddelbart tenker jeg at dette kan være en case for en custom komponent. Vi kunne hatt en komponent som het noe sånn som TextFieldForDarkBakground? Så bruker vi heller den der vi bruker mørk bakgrunnsfarge, og ellers bruker vi MUI sin standard TextField? Evt kan vi lage en egen CustomTextField som sjekker bakgrunnsfargen og gir oss enten TextField eller TextFieldForDarkBakground basert på lys/mørk bakgrunn?

Reply:
hmm, CustomTextField høres ikke så dumt ut... Blir i så fall det samme for flere typer også da, CustomAutocomplete, (CustomRadio?), osv

Reply:
ja, men tror vi kommer til å tjene på det i fremtiden allikevel. Det kommer jo enda flere nye typer etterhvert også